client/src/app/components/environments/environment-list/environment-list-view.component.html

```diff
@@ -0,0 +1,75 @@
+<div class="flex items-center justify-between mb-3">
+  <input pInputText id="commit-hash" (input)="onSearch($event)" [value]="searchInput()" type="text" placeholder="Search for installed systems" class="w-1/3" />
+  @if (!hideLinkToList()) {
+    <a [routerLink]="'/environments'" class="p-button p-2 p-button-secondary self-end">Manage environments</a>
+  }
+</div>
+
+<p-accordion [multiple]="true">
+  @if (!query.isPending() && filteredEnvironments().length === 0) {
+    @if (query.isError()) {
+      <p>Failed to load environments. Please try again later.</p>
+    } @else if (searchInput().length === 0) {
+      <p>No environments found. Create a new environment to get started.</p>
+    } @else {
+      <p>No environments found for the search term, try something else.</p>
+    }
+  }
+
+  @for (environment of filteredEnvironments(); track environment.id) {
+    <p-accordionTab>
+      <ng-template pTemplate="header">
+        <div class="flex gap-2 items-center w-full">
+          <span>{{ environment.name }}</span>
+
+          <app-lock-tag [isLocked]="!!environment.locked"></app-lock-tag>
+
+          @if (environment.latestDeployment; as deployment) {
+            <app-deployment-state-tag [state]="deployment.state || 'IN_PROGRESS'" />
+          }
+
+          <div class="flex-grow"></div>
+
+          @if (!environment.locked && deployable()) {
+            <button (click)="deployEnvironment(environment); $event.stopPropagation()" class="p-button p-button-secondary p-2">
+              <i-tabler name="cloud-upload" class="mr-1" />Deploy
+            </button>
+          }
+
+          @if (environment.locked) {
+            @if (environment.serverUrl) {
+              <a [href]="getFullUrl(environment.serverUrl)" target="_blank" class="p-button p-button-secondary p-2" (click)="$event.stopPropagation()"
+                ><i-tabler name="external-link" class="mr-1" />Open</a
+              >
+            }
+            <button (click)="unlockEnvironment.mutate(environment); $event.stopPropagation()" class="p-button p-button-danger p-2">
+              <i-tabler name="lock-open" class="mr-1" />Unlock
+            </button>
+          }
+
```

Comment:
Looks like you’ve removed the ‘Open link’ button from environment list view. Can we add it back. Since we are currently allowing users to configure URL for each environments on edit.